documentation/docs/admin/production-setup/multiple-artemis-instances.mdx

```diff
@@ -0,0 +1,770 @@
+to create this cluster.
+This is problematic if the instances change dynamically.
+Artemis uses a discovery service to solve the issue (named [JHipster Registry](https://www.jhipster.tech/jhipster-registry/)).
+
+
+### Discovery service
+
+JHipster registry contains Eureka, the discovery service where all instances can register themselves and fetch
+the other registered instances.
+
+Eureka can be configured like this within Artemis:
+
+```yaml
+# Eureka configuration
+eureka:
+    client:
+        enabled: true
+        service-url:
+            defaultZone: {{ artemis_eureka_urls }}
+instance:
+    prefer-ip-address: true
+    ip-address: {{ artemis_ip_address }}
+    appname: Artemis
+    instanceId: Artemis:{{ artemis_eureka_instance_id }}
+
+logging:
+    file:
+        name: '/opt/artemis/artemis.log'
+```
+
+`{{ artemis_eureka_urls }}` must be the URL where Eureka is reachable,
+`{{ artemis_ip_address }}` must be the IP under which this instance is reachable and
+`{{ artemis_eureka_instance_id }}` must be a unique identifier for this instance.
+You also have to set up the value `jhipster.registry.password` to the password of the registry
+(which you will set later).
+
+Note that Hazelcast (which requires Eureka) is by default binding to `127.0.0.1` to prevent other instances
+to form a cluster without manual intervention.
+If you set up the cluster on multiple machines (which you should do for a production setup),
+you have to set the value `spring.hazelcast.interface` to the ip-address of the machine.
+Hazelcast will then bind on this interface rather than `127.0.0.1`,
+which allows other instances to establish connections to the instance.
+This setting must be set for every instance, but you have to make sure to adjust the ip-address correspondingly.
+
+
+#### Setup
+
+##### Installation
+
+1. Create the directories `/opt/registry/` and `/opt/registry/config-server`.
+
+2. Download the application
+
+    Download the latest version of the jhipster-registry JAR from GitHub: https://github.com/jhipster/jhipster-registry/releases/.
+
+
+##### Service configuration
+
+1. Configure `/etc/systemd/system/registry.service`:
+
+    ```systemd
+    [Unit]
+    Description=Registry
+    After=syslog.target
+
+    [Service]
+    User=artemis
+    WorkingDirectory=/opt/registry
+    ExecStart=/usr/bin/java \
+        -Xmx256m \
+        -jar registry.jar \
+        --spring.profiles.active=prod,native
+    SuccessExitStatus=143
+    StandardOutput=/opt/registry/registry.log
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+2. Set Permissions in Registry Folder
+
+    ```bash
+    chown -R artemis:artemis /opt/registry
+    chmod g+rwx /opt/registry
+    ```
+
+3. Enable the service
+
+    ```bash
+    systemctl daemon-reload
+    systemctl enable registry.service
+    ```
+
+4. Start Service
+
+    ```bash
+    systemctl start registry
+    ```
+
```

Comment:
Why did we drop the logging command here? 

`sudo journalctl -f -n 1000 -u registry` ? 

<img width="985" height="245" alt="image" src="https://github.com/user-attachments/assets/c0b2e84d-fe8e-4433-8282-4a2e5826abba" />